src/globus_sdk/_testing/data/search/reopen_index.py

```diff
@@ -0,0 +1,19 @@
+import uuid
+
+from globus_sdk._testing.models import RegisteredResponse, ResponseSet
+
+INDEX_ID = str(uuid.uuid4())
+
+
+RESPONSES = ResponseSet(
+    default=RegisteredResponse(
+        service="search",
+        method="POST",
+        path=f"/v1/index/{INDEX_ID}/reopen",
+        json={
+            "index_id": INDEX_ID,
+            "acknowledged": True,
+        },
+        metadata={"index_id": INDEX_ID},
+    )
+)
```

Comment:
Is there a different response that might be returned for these scenarios:

* The index has not been marked for deletion
* The index is trial and within the brief window where it hasn't been deleted yet
* The index has already been deleted

Reply:
Yeah, there are a few failure modes here, for all of these APIs.
Index create can hit your trial limit, delete can 404, etc.

I can add some failures into the dataset.